src/core/lib/iomgr/event_engine_shims/resolved_address.cc

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include <grpc/support/port_platform.h>
+
+#include "src/core/lib/iomgr/event_engine_shims/resolved_address.h"
+
+#include <grpc/event_engine/event_engine.h>
+
+#include "src/core/lib/iomgr/resolved_address.h"
+
+namespace grpc_event_engine {
+namespace experimental {
+
+EventEngine::ResolvedAddress CreateResolvedAddress(
```

Comment:
I think we _may_ need this later, after the shims are gone. Can we move these conversion methods to `src/core/lib/event_engine/resolved_address_internal.h`, and keep the implementations with the rest of `src/core/lib/event_engine/resolved_address.cc`?

Reply:
Moved it now.